src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -191,6 +188,9 @@
     <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.LookupAccountNameW.cs">
       <Link>Common\Interop\Windows\mincore\Interop.LookupAccountNameW.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.SHGetKnownFolderPath.cs">
```

Comment:
It should be moved into `Condition=" '$(TargetsWindows)' == 'true' And '$(CoreClrOrCorRt)' == 'true'` block instead.